docs/api/php_api/php_api.md

```diff
@@ -37,6 +38,7 @@ The full list of available services covers:
 - [SearchService](https://github.com/ibexa/core/blob/main/src/contracts/Repository/SearchService.php)
 - [SectionService](https://github.com/ibexa/core/blob/main/src/contracts/Repository/SectionService.php)
 - ShippingAddressService
+- SpreadsheetProcessorService
```

Comment:
Which exactly class you have mind here ?

Reply:
`Ibexa\Cart\FileProcessor\SpreadsheetProcessor`

Reply:
So should I change the name to SpreadsheetProcessor or remove it?

Reply:
Then I believe we should mention  `\Ibexa\Contracts\Cart\FileProcessor\SpreadsheetProcessorInterface` as `Ibexa\Cart\FileProcessor\SpreadsheetProcessor` is specific implementation of it